lms/static/styles/components/_FullScreenSpinner.scss

```diff
@@ -0,0 +1,18 @@
+@use "sass:math";
+
+@use "../mixins";
+@use "../variables" as var;
+
+.FullScreenSpinner__backdrop {
+  background-color: white;
+  bottom: 0;
```

Comment:
nit: bottom, left, right and top could be all together
something like this..
```
background-color: white;
opacity: 0.5;  
position: fixed;
left: 0;
top: 0
right: 0
bottom: 0;

```

Reply:
Agreed. The properties were previously sorted alphabetically but it makes sense to group related properties.